In [1]:
import pandas as pd
import numpy as np

In [2]:
iris = pd.read_csv('iris.csv')
iris

sepal.length  sepal.width  petal.length  petal.width    variety
0             5.1          3.5           1.4          0.2     Setosa
1             4.9          3.0           1.4          0.2     Setosa
2             4.7          3.2           1.3          0.2     Setosa
3             4.6          3.1           1.5          0.2     Setosa
4             5.0          3.6           1.4          0.2     Setosa
..            ...          ...           ...          ...        ...
145           6.7          3.0           5.2          2.3  Virginica
146           6.3          2.5           5.0          1.9  Virginica
147           6.5          3.0           5.2          2.0  Virginica
148           6.2          3.4           5.4          2.3  Virginica
149           5.9          3.0           5.1          1.8  Virginica

[150 rows x 5 columns]

In [4]:
iris.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 150 entries, 0 to 149
Data columns (total 5 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   sepal.length  150 non-null    float64
 1   sepal.width   150 non-null    float64
 2   petal.length  150 non-null    float64
 3   petal.width   150 non-null    float64
 4   variety       150 non-null    object 
dtypes: float64(4), object(1)
memory usage: 6.0+ KB


In [7]:
missing_values=iris.isna().sum()
missing_values

sepal.length    0
sepal.width     0
petal.length    0
petal.width     0
variety         0
dtype: int64

In [9]:
duplicated_values = iris.duplicated().sum()
duplicated_values

1

In [10]:
iris1 = iris.drop_duplicated()
print(iris1)

AttributeError: 'DataFrame' object has no attribute 'drop_duplicated'

# Observations
- There are 150 rows and 5 columns
- There are no Null values
- There are one duplicated row
- x columns are sepal.length,sepal.width,petal.length and petal.width
- All the x-columns are continuous
- The y-column is "variety" which is categorical